Pseudo/css/pseudoClass.css

```diff
@@ -0,0 +1,57 @@
+h3 {
+	font-size: 20px;
+	line-height: 22px;
+	margin-bottom: 8px;
+}
+.section {
+	margin-bottom: 20px;
+}
+
+/*anchorStyling*/
+
+.anchorStyling a {
+	font-size: 16px;
+	line-height: 18px;
+}
+.anchorStyling a:link {
+	color: blue;
```

Comment:
Use Hex Colors everywhere.
Refer to this article for reading how to utilise devtools for converting between rgb, hex, hsla color conventions.
https://www.keycdn.com/blog/chrome-devtools/

Reply:
Done